dftool

```diff
@@ -3,7 +3,7 @@
 import datetime
 import os
 import sys
-import functools
```

Comment:
There are a _ton_ of changes in dftool that have nothing to do with the main goal of this PR. It's super-hard for me to separate as I review.

Therefore as above: please move changes that are outside the scope of this PR's goal to >=1 new PRs. 

When you're done, please request a new review from me for this PR (and the other PRs).

Reply:
Done.

Reply:
Ok, I see what happened... 

I'm gonna open a separate PR `issue270-new-dftool-commands` and move these commands there.
We can then close this PR/branch so nothing dirty makes it in.